packages/app-page-builder/src/editor/plugins/elements/media/iframe/IFrame.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { ReactComponent as IFrameIcon } from "./iframe-icon.svg";
+import { PbEditorElement } from "~/types";
+import { PageBuilderContext } from "../../../../../contexts/PageBuilder";
```

Comment:
You can use the `~/` to point to the root of the package. See the `~/types` import above.